### PR TITLE
Added volume snapshot querying

### PR DESCRIPTION
**Pull request by jdenoy**
Added volume snapshot querying.

OVH API : https://api.ovh.com/console/#/cloud/project/{serviceName}/volume/snapshot~GET

### CODE REVIEW

File: ovh/plugin.go [hunk 1]
```diff
@@ -16,21 +16,22 @@ func Plugin(ctx context.Context) *plugin.Plugin {
...
+			"ovh_cloud_database":    		tableOvhCloudDatabase(),
+			"ovh_cloud_flavor":      		tableOvhCloudFlavor(),
+			"ovh_cloud_image":       		tableOvhCloudImage(),
+			"ovh_cloud_instance":    		tableOvhCloudInstance(),
+			"ovh_cloud_postgres":    		tableOvhCloudPostgres(),
+			"ovh_cloud_project":     		tableOvhCloudProject(),
+			"ovh_cloud_ssh_key":     		tableOvhCloudSshKey(),
+			"ovh_cloud_storage":     		tableOvhCloudStorage(),
+			"ovh_cloud_volume":      		tableOvhCloudVolume(),
+			"ovh_cloud_volume_snapshot": 	tableOvhCloudVolumeSnapshot(),
```
Comment (francois2metz) [inline ovh/plugin.go:34]: Could you fix the indentation here?

File: ovh/table_ovh_cloud_volume_snapshot.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableOvhCloudVolumeSnapshot() *plugin.Table {
+	return &plugin.Table{
+		Name:        "ovh_cloud_volume_snapshot",
+		Description: "A volume is an independent additional disk.",
```
Comment (francois2metz) [inline ovh/table_ovh_cloud_volume_snapshot.go:16]: The description should be updated I guess.

File: ovh/table_ovh_cloud_volume_snapshot.go [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+			{
+				Name:        "project_id",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromQual("project_id"),
+				Description: "Project ID.",
+			},
+			{
+				Name:        "id",
+				Type:        proto.ColumnType_STRING,
+				Description: "ID.",
```
Comment (francois2metz) [inline ovh/table_ovh_cloud_volume_snapshot.go:35]:
```suggestion
				Description: "Snapshot ID.",
```

File: ovh/table_ovh_cloud_volume_snapshot.go [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+			},
+			{
+				Name:        "creationDate",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "Volume creation date.",
```
Comment (francois2metz) [inline ovh/table_ovh_cloud_volume_snapshot.go:40]:
```suggestion
				Description: "Snapshot creation date.",
```

File: ovh/table_ovh_cloud_volume_snapshot.go [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+			},
+			{
+				Name:        "size",
+				Type:        proto.ColumnType_INT,
+				Description: "Volume Snapshot size (in GB).",
+			},
+			{
+				Name:        "volumeId",
+				Type:        proto.ColumnType_STRING,
+				Description: "Volume Snapshot ID.",
```
Comment (francois2metz) [inline ovh/table_ovh_cloud_volume_snapshot.go:61]:
```suggestion
				Description: "Volume source ID.",
```

File: ovh/table_ovh_cloud_volume_snapshot.go [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+			},
+			{
+				Name:        "region",
+				Type:        proto.ColumnType_STRING,
+				Description: "Volume Snapshot Region.",
+			},
+			{
+				Name:        "status",
+				Type:        proto.ColumnType_STRING,
+				Description: "Volume Snapshot Status. (available, creating, deleting, error, error_deleting)",
```
Comment (francois2metz) [inline ovh/table_ovh_cloud_volume_snapshot.go:72]:
```suggestion
				Description: "Volume Snapshot Status (available, creating, deleting, error, error_deleting).",
```

**Comment by francois2metz** [comment]
Btw thanks for your contribution! I appreciate it.

**Comment by jdenoy** [comment]
Will send new PR with corrections

**Pull request closed without merging**